.github/workflows/automate_javadoc.yml

```diff
@@ -1,27 +1,29 @@
-# SPDX-FileCopyrightText: 2022 RTE FRANCE
+# SPDX-FileCopyrightText: 2022 2023 RTE FRANCE
 #
 # SPDX-License-Identifier: Apache-2.0
 name: Continuous Deployment - JavaDocs
 
 on:
-  push:
-    branches:
-      - 'feat/javadoc'
+  pull_request_target:
+    types: [ labeled, closed ]
```

Comment:
Hi @AliouDIAITE 

I would like you to read this PR.

Whenever a developer creates a pull request from any branch to `develop`, he must have already set the label `javadoc` or `documentation`.

After her PR is merged, `automate_javadoc.yml ` is triggered, which leads to the creation of a new PR from _depondabot_ to the **gh-pages** branch and it's automatically assigned to the author of the merged one.

Then the gh-pages deployment is ready to update javadoc once he accepts the _depondabot_ PR.

I also mentioned that `gh-pages` actually doesn't have a documentation folder (docs), this need #228